### ISSUE TITLE
Copy files before compiling assets

### ISSUE THREAD

**Issue by AndreRudolph** [symfony/webpack-encore#1065]
I have a scenario that I need to copy (ES6 source files) files from a bundle into my symfony projects assets/ directory in order to import them in my projects assets. However I use the copyFiles method below: 

```
    .copyFiles({
        from: './vendor/path/to/bundle/src/Resources/assets/js/',
        to: './../../assets/js/bundlename/[path]/[name].[ext]'
    })
```

The copy process in general works fine but on the first run the imported js resources in my symfony applications assets can not be found. I guess the copying happens during the build (npm run dev) simultaneousely when the assets will be compiled or after that. For the second run it works fine since the copied assets are in place. Is there any other option to copy files in the execution cycle **_before_** the assets will be compiled?

Thanks in advance.

**Comment by stof** [comment]
Why not importing them from the vendor folder (maybe using an alias to have a simple path) ? Copying files is meant to handle the case of copying files to the build output to reference them from your templates.

**Comment by AndreRudolph** [comment]
Well, I already tried that but then I was getting errors that underscore and axios referenced files were not found. But while I tried the approach in the previous post I disabled the symlinks resolve like this

```
const config = Encore.getWebpackConfig();
config.resolve.symlinks = false;
```

and now I tried it again with just importing the files and  it seems to work now with imports as I intentionally wanted. So thanks for your answer.
